Sources/ArgumentDescription.swift

```diff
@@ -152,6 +152,37 @@ open class Options<T : ArgumentConvertible> : ArgumentDescriptor {
 }
 
 
+open class VaradicOption<T : ArgumentConvertible> : ArgumentDescriptor {
+  public typealias ValueType = [T]
+
+  open let name:String
+  open let description:String?
+  open let `default`:ValueType
+  open var type:ArgumentType { return .option }
+
+  public init(_ name:String, _ default:ValueType, description:String? = nil) {
```

Comment:
Since the value is now an array, it might make sense to make the `default` value optional and default to an empty array.